src/table/sticky-scrollbar/use-sticky-scrollbar.ts

```diff
@@ -109,7 +110,15 @@ export function useStickyScrollbar(
         observer.disconnect();
       };
     }
-  }, [scrollbarContentRef, scrollbarRef, tableRef, wrapperRef, consideredFooterHeight, hasContainingBlock]);
+  }, [
+    scrollbarContentRef,
+    scrollbarRef,
+    tableRef,
+    wrapperRef,
+    consideredFooterHeight,
+    hasContainingBlock,
+    offsetScrollbar,
```

Comment:
Adding this `offsetScrollbar` dependency will make the computation re-run, fixing the issue.